test/metabase/driver/sql_jdbc_test.clj

```diff
@@ -282,3 +285,18 @@
               (let [syncable (driver/syncable-schemas driver/*driver* db-filtered)]
                 (is (not (contains? syncable "public")))
                 (is (not (contains? syncable fake-schema-name)))))))))))
+
+(deftest sync-estimated-row-count-test
```

Comment:
The previous comment got lost because the code was removed, so here it is
https://github.com/metabase/metabase/pull/39494/files/5e15c627f41d4fa81f89680ab05cae675f745828#r1534344084

I mentioned this should be in a sync test namespace, but this is in a driver test namespace. If we're to take modularization seriously there shouldn't be any sync related code in this namespace.

Maybe a good place to put this test is `metabase.sync.sync-metadata.tables-test`?

It's unfortunate this test uses `(next.jdbc/execute! conn ["VACUUM;"])` which is postgres specific but I think we can worry about that when we get to having multiple drivers support this feature

Reply:
Perhaps vacuum (or analyze) could be put behind a test multimethod (something like `update-table-statistics!`)

Reply:
Moved to metabase.sync.sync-metadata.tables-test

on making vacuum a multimethod: absolutely, but I'll get to it once we add more driver supports :)